owa/app/js/components/manage_patient_identifier_sources/view_patient_identifier_sources/localIdentifierGenerator.jsx

```diff
@@ -178,15 +177,15 @@ export default class ViewLocalIdentifierGenerator extends React.Component {
               style={{
                 textAlign: 'center', 
                 height: '60px', 
-                width: '100%', 
+                width: '95%', 
                 padding: '5px', 
                 margin: '5px', 
                 border: '2px dashed #000'}} 
               onDrop={this.onDrop.bind(this)}>
                 <p>{this.state.fileName}</p>
               </Dropzone>
             </div>
-            <input type="submit" onClick={this.handleAddReservedIdentifiers} value="Upload" />
+            <input type="submit" style={{marginLeft: '40%'}} onClick={this.handleAddReservedIdentifiers} value="Upload" />
```

Comment:
Curious question: Why did you opt to handle the submit with `input` vs. handling it with a `button`?

Reply:
@LWanjiru For consistency and I only need text to be displayed on the button.

Reply:
👍